File: TP/TP1/Code/5J-3.cpp
/* Afficher un motif donné */
#include <iostream>

int main()
{
  for (int i = 0; i < 7; ++i) {
    for (int j = 0; j < 9; ++j) {
      if (j == 4 || i == 3)
        std::cout << "*" << std::flush;
      else
        std::cout << " " << std::flush;
    }
    std::cout << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/5J-3.py
#!/usr/bin/python
# Afficher un motif donné
for i in range(7):
    for j in range(9):
        if j == 4 or i == 3:
            print('*', end='')
        else:
            print(' ', end='')
    print('')  # Ici on affiche un saut de ligne
